pkg/machine/qemu/machine.go

```diff
@@ -1126,8 +1126,8 @@ func (v *MachineVM) startHostNetworking() (string, machine.APIForwardingState, e
 		cmd.Debug = true
 		logrus.Debug(cmd)
 	}
-
 	c := cmd.Cmd(binary)
+	logrus.Debug(c.Args)
```

Comment:
Extra debug?

Reply:
no, i think this is good thing?

Reply:
I'm OK with that but add a message about what's being printed so it's more obvious what's going on?